docs/90-FAQ.md

```diff
@@ -283,6 +291,18 @@ you don't want new clients to be urgent by default put this in your rc.lua:
         end
     end
     client.connect_signal("request::activate", awful.ewmh.activate)
+    
+### How to prevent mouse cursor from jumping to the corner when resizing or moving a client
+
+You can specify the behavior for these layouts:
+
+    awful.layout.suit.floating.resize_jump_to_corner = false
+    awful.layout.suit.tile.resize_jump_to_corner = false
```

Comment:
`awful/mouse/init.lua`, line 213 need to be adapted with offsets like the one and pretends like like the one just above in `.move()`. It currently isn't exposed externally, so it will need a config option.